src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java

```diff
@@ -7,11 +7,9 @@
 import static org.nebula.contrib.ngbatis.utils.ReflectUtil.castNumber;
 import static org.nebula.contrib.ngbatis.utils.ReflectUtil.getPkField;
 
-import com.vesoft.nebula.client.graph.data.DateTimeWrapper;
-import com.vesoft.nebula.client.graph.data.DateWrapper;
-import com.vesoft.nebula.client.graph.data.Node;
-import com.vesoft.nebula.client.graph.data.Relationship;
-import com.vesoft.nebula.client.graph.data.ValueWrapper;
+import com.vesoft.nebula.ErrorCode;
+import com.vesoft.nebula.client.graph.data.*;

```

Comment:
It seems that stylecheck does not allow `*` to replace various classes

Reply:
yes, you are right. the import in the form of ". *" has been fixed